go.mod

```diff
@@ -23,13 +23,13 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/sergi/go-diff v1.0.0
 	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.4.0
+	github.com/stretchr/testify v1.5.1
 	golang.org/x/net v0.0.0-20200202094626-16171245cfb2 // indirect
 	golang.org/x/sys v0.0.0-20200202164722-d101bd2416d5
 	golang.org/x/text v0.3.2 // indirect
-	google.golang.org/grpc v1.27.1
+	google.golang.org/grpc v1.38.0
```

Comment:
we've had some issues in the past with the protobuf like https://github.com/kubernetes-csi/csi-proxy/issues/159, did you run the integration tests?

Also I believe the client module should be updated too https://github.com/kubernetes-csi/csi-proxy/blob/master/client/go.mod

Reply:
@mauriciopoppe updated the clients versions too. btw the protobuf carry the version which has the fix as mentioned in referenced issue.  Reg# integration tests, I was in an impression that, CI ( windows test/integration) cover it. ptal , thanks

Reply:
great, thanks for updating the client module too

some of the integration tests can't run in the CI environment provided by Github because it's a constrained environment that doesn't allow some privileged storage operations so we had to run them manually in a GCE Windows VM